### ISSUE TITLE
false positive, class attribute names

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#312]
when part of BEGIN OF,

![image](https://user-images.githubusercontent.com/5888506/50539015-acd37b80-0b79-11e9-95bc-62e9e2555fd9.png)

```abap
  PROTECTED SECTION.

    TYPES:
      ty_byte TYPE x LENGTH 1 .

    DATA:
      BEGIN OF ms_message,
        received TYPE abap_bool,
        message  TYPE x LENGTH 1,
      END OF ms_message .
```

**Comment by larshp** [comment]
this might have been fixed, to be tested

**Comment by larshp** [comment]
![image](https://user-images.githubusercontent.com/5888506/76250245-3e8e4c80-6245-11ea-9e25-28e4f7e74bad.png)

ok, closing
